packages/shopify-app-remix/docs/staticPages/billing.doc.ts

```diff
@@ -0,0 +1,89 @@
+import {LandingTemplateSchema} from '@shopify/generate-docs';
+
+const data: LandingTemplateSchema = {
+  id: 'guide-billing',
+  title: 'Billing for your app',
+  description:
+    'You can bill merchants for your app via Shopify using the Admin API, either as a one-time purchase or as a recurring subscription.' +
+    '\n\nShopify will handle the transactions, based on purchases set up by the app and agreed upon by the merchant.' +
+    '\n\nFor more information on how this process works, see [the billing documentation](/docs/apps/billing).',
+  sections: [
+    {
+      type: 'Generic',
+      anchorLink: 'plans',
+      title: 'Setting up plans',
+      sectionContent:
+        'Your app can offer one or more plans to merchants. These plans can be one-time purchases or recurring subscriptions.' +
+        '\n\nTo configure a plan, use the `billing` configuration when calling `shopifyApp`. This configuration is a hash, where the keys are the plan name that will appear in the purchase, and the values are the plan settings.',
```

Comment:
Should we go into depth here on the billing config object? The keys and types? Or link to other docs if it will be documented elsewhere